### PR TITLE
Insub migration for database

### PR DESCRIPTION
**Pull request by cuongnguyen71190**
please help me review it.

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -13,7 +14,7 @@ class AppServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        //
+        Schema::defaultStringLength(191);
```
Comment (tucq88) [inline app/Providers/AppServiceProvider.php]: @cuongnguyen71190 Why need this ? Encrypted password may over 191 chars

Comment (cuongnguyen71190) [inline app/Providers/AppServiceProvider.php]: when i register user, on screen show an error about database  or something (i don't remember). but i search google and there is a answer above

Comment (cuongnguyen71190) [inline app/Providers/AppServiceProvider.php]: it here: https://laravel-news.com/laravel-5-4-key-too-long-error

Comment (tucq88) [inline app/Providers/AppServiceProvider.php]: oh shit !

**Pull request merged**